transformations/wordstonumbers/transformation.py

```diff
@@ -0,0 +1,18 @@
+from typing import List
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+from text2nums import *
+
+
+class WordsToNumbers(SentenceOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION, TaskType.TEXT_TAGGING]
```

Comment:
Task TEXT_TAGGING is not applicable here because of a change in the number of words (Ex: I have two hundred fifty books --> I have 250 books.)

Reply:
You can add tasks PARAPHRASE_DETECTION, TEXTUAL_ENTAILMENT